src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -3,12 +3,10 @@
     xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
     xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd
```

Comment:
you need to revert this, otherwise it will fail when we'll deploy it in prod

Reply:
Are you sure about it? I think an update to XML schema location will not impact existing migrations.

Reply:
I'm afraid it will change the checksum in database, so the app won't start

Reply:
Per liquibase docs, schema location of liquibase dependency is not part of checksum. 
>Liquibase will insert a new row with the id/author/filepath along with an MD5Sum of the changeset

Reply:
thanks a lot, I didn't know it !